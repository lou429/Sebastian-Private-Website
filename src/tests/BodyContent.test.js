import BodyComponent from '../components/BodyContent';
import { render } from '@testing-library/react';

test('Body content can be rendered', () => {
    let text = "Test render"; 

    const { container, getByText } = render(
        <BodyComponent>
            <p>
                {text}
            </p>
        </BodyComponent>
    );
    
    expect(getByText(text)).toBeInTheDocument();
    expect(container).toMatchSnapshot();
});