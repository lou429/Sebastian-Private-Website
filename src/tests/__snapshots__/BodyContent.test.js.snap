// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body content can be rendered 1`] = `
<div>
  <div
    class="content-body"
  >
    <p>
      Test render
    </p>
  </div>
</div>
`;
